tests/emulators/ibmfakebelem_q01/parameters.json

```diff
@@ -0,0 +1,280 @@
+{
```

Comment:
Yes this was previously only there for some tests. Now that everything is settled, I have added the platform.py file in  bd5772e56695ba932389563636dd2c867a0b91fe. Note that if you do wish to try it, the "CZ" gate is actually a "CX" gate. Can I set "CX" gate as a native gate in the current version of qibolab? I remember this was not possible in an earlier version and that's why I just called it a "CZ" gate as a temporary hack.

Reply:
> Can I set "CX" gate as a native gate in the current version of qibolab?

In the `TwoQubitNatives` class there is a `CNOT`: https://github.com/qiboteam/qibolab/blob/7868bfe81c7a11802f975fcbca962fb2f02f5647/src/qibolab/native.py#L350
which should be equivalent to the CX. So you should be able to register `"CNOT"` gates in the parameters.json. Maybe it is better to use that, instead of CZ?